crates/bevy_asset/src/io/file_asset_io.rs

```diff
@@ -64,7 +64,14 @@ impl FileAssetIo {
     /// instead. It's set by cargo when running with `cargo run`.
     pub fn get_base_path() -> PathBuf {
         if let Ok(manifest_dir) = env::var("CARGO_MANIFEST_DIR") {
-            PathBuf::from(manifest_dir)
+            // Some Windows software don't support canonicalized path names, so let's avoid them
+            // unless the path is relative, in which case we currently need to make it absolute
+            // (See more: https://github.com/rust-lang/rust/issues/59117 )
+            if Path::new(&manifest_dir).is_relative() {
```

Comment:
I don't think overloading `CARGO_MANIFEST_DIR` with the responsibility of "configuring deployed app asset folder root relative paths" is the right path forward, especially if it is consistently an absolute path within the context of cargo. At the very least, we should have a new environment variable for this scenario.

However, I don't think we should be encouraging "user-configurable" asset roots at all by default. Asset path logic should be determined by the app developer, not the app consumer. The defaults will work in 99.99% of cases and forcing standardization here (by default) is valuable from a tooling and ecosystem perspective.

If you want your game to support non-standard user-configurable asset paths, I think the correct solution is to use `AssetServerSettings::asset_folder` (and maybe add a configurable `custom_root_path: Option<PathBuf>` to the settings). You could wire that up to whatever environment variable you choose.